cmd/ff-proxy/main.go

```diff
@@ -359,6 +361,23 @@ func main() {
 
 		logger.Info("retrieved offline config")
 	} else {
+		gpc := gripcontrol.NewGripPubControl([]map[string]interface{}{
+			{
+				"control_uri": "http://localhost:5561",
+			},
+		})
+
+		var eventListener sdkStream.EventStreamListener
```

Comment:
I see on [line 417](https://github.com/harness/ff-proxy/pull/99/files#diff-4ffaa8cb395a69efc0cd1ee9a7c1459387c4b3d3731d2519c82a27f640c671b9R417) this gets passed to the internal SDK startup code. 

If the stream healthcheck fails I think this eventListener will be nil? Just wondering if that could cause any issues with the internal SDK and if we should maybe implement a NoOp EventListener that we assign to this if the healthcheck fails?

Reply:
Yes I'd dug into that myself, the go sdk does only forward events to that `Pub` function if the eventListener isn't nil so it should already be a no op if it's not set from what I can see 
https://github.com/harness/ff-golang-server-sdk/blob/main/stream/sse.go#L169

Reply:
Ahh cool, should be sweet then